content/en-us/reference/engine/classes/AudioAnalyzer.yaml

```diff
@@ -117,6 +117,7 @@ methods:
       Returns the frequency spectrum of the last audio buffer, as an array of
       numbers. The elements of the array are root-mean-square volume levels,
       evenly spaced from 0 hertz to 24,000 hertz.
+      If any of the analyzer's inputs come from an `Class.AudioDeviceInput`, this method returns an empty array.
```

Comment:
We also have this disabled for `AudioEcho`, right? Do we need to mention that as well

Reply:
All the exceptions are logically downstream from AudioDeviceInput

Reply:
I think the relationship is less obvious if it's an AudioEcho or AudioReverb that has never been wired to an AudioDeviceInput before